pkg/qemu/qemu.go

```diff
@@ -500,28 +451,26 @@ func Cmdline(ctx context.Context, cfg Config) (exe string, args []string, err er
 		if version.LessThan(*semver.New(MinimumQemuVersion)) {
 			logrus.Fatalf("QEMU %v is too old, %v or later required", version, MinimumQemuVersion)
 		}
+		if version.LessThan(*semver.New(RecommendedQemuVersion)) {
+			logrus.Warnf("QEMU %v is too old, %v or later is recommended", version, RecommendedQemuVersion)
+		}
 		if y.VMOpts.QEMU.MinimumVersion != nil && version.LessThan(*semver.New(*y.VMOpts.QEMU.MinimumVersion)) {
 			logrus.Fatalf("QEMU %v is too old, template requires %q or later", version, *y.VMOpts.QEMU.MinimumVersion)
 		}
-		if runtime.GOOS == "darwin" && runtime.GOARCH == "arm64" && version.Equal(*semver.New("8.2.0")) {
```

Comment:
Is there a reason to remove this warning? It is unlikely it will ever be triggered, but checks for a bug in a QEMU version higher than the recommended version.

Reply:
Maybe we should just set `RecommendedQemuVersion = "8.2.1"` in `qemu_darwin.go`?

Reply:
Fine with me

Reply:
I'm sorry, I still think this warning should stay unless you set `MinimumVersion` to `8.2.1` on macOS. I think this would be fine to do; it is just Linux that may need ancient QEMU versions.

Reply:
Updated to set minimum to 8.2.1 for arm64